RadixWallet/Features/TransactionReviewFeature/TransactionReview.swift

```diff
@@ -358,6 +358,12 @@ public struct TransactionReview: Sendable, FeatureReducer {
 					.init(
 						resourceAddress: transfer.resource.resourceAddress,
 						resource: .success(transfer.resource),
+						ownedFungibleResource: .init(
```

Comment:
I am still hoping that one day we will get rid of the "owned" prefixes but I guess today isn't the day...